token-metadata/program/tests/print.rs

```diff
@@ -0,0 +1,456 @@
+#![cfg(feature = "test-bpf")]
+pub mod utils;
+
+use borsh::BorshSerialize;
+use mpl_token_metadata::{
+    error::MetadataError,
+    state::{Key, MAX_MASTER_EDITION_LEN},
+};
+use num_traits::FromPrimitive;
+use solana_program_test::*;
+use solana_sdk::{
+    account::AccountSharedData, instruction::InstructionError, transaction::TransactionError,
+};
+use utils::*;
+
+// NOTE: these tests depend on the token-vault program having been compiled
```

Comment:
This is copy paste and the comment can be removed both here and in the original as Token Vault is deprecated and no longer a required dependency.